modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -291,6 +291,13 @@ export default class TileHeader {
     return this._distanceToCamera;
   }
 
+  /**
+   * Screen space error for LOD selection
+   */
+  get screenSpaceError(): number {
```

Comment:
Nit: isn't it really confusing for the user to have both a `screenSpaceError` getter and a `getScreenSpaceError()` method that does totally different things?

Does it have to be a getter? Can we clarify things with naming?
```suggestion
  getScreenSpaceErrorForLODSelection(): number {
```

Reply:
It is getter because it returns internal value of the corresponding private property that was calculated before. `screenSpaceError` is always for LOD selection. I left a comment but don't see why we need to mention it in the name.
Method `getScreenSpaceError` is to calculate `screenSpaceError` with explicit viewport parameters.